src/google/protobuf/compiler/csharp/csharp_generator.cc

```diff
@@ -28,7 +28,13 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+// Authors: Jan Tattermusch <jtattermusch@google.com>
```

Comment:
Hmmm.... it was mostly from looking at other files. Some [files](https://github.com/google/protobuf/blob/master/src/google/protobuf/compiler/cpp/cpp_generator.cc#L31-L33) that I saw had them so I punched as well.